### DISCUSSION TITLE
Creating Windows build in Linux Docker container

### DISCUSSION THREAD

**Discussion by WolfBublitz** [AvaloniaUI/Avalonia#7313] [Q&A]
Hallo,

I am currently facing the following situation:

I am developing an Avalonia application on Windows (with .NET 6) and in the csproj file I have the following option enabled

`<OutputType>WinExe</OutputType>`

to prevent the application to start with a terminal window on Windows. This works as expected.

Now when the application is built inside a Linux Docker container by running the following command

`dotnet publish --configuration Release --self-contained=true /p:PublishSingleFile=true --runtime win-x64`

I get a working Windows executable, but it starts again with a terminal window popping up at first. As far as I see it the 

`<OutputType>WinExe</OutputType>`

option does not work while building a Windows binary on Linux.

My Question: Is there a solution to build a Windows binary on Linux that does not pop up the terminal on start?

Best regards!

**Comment by jp2masa** [comment]
#6672
